mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -399,14 +421,19 @@ class WidgetAdapter(
         inflater: LayoutInflater,
         parent: ViewGroup,
         private val connection: Connection,
-        colorMapper: ColorMapper
+        colorMapper: ColorMapper,
+        context: Context
     ) : LabeledItemBaseViewHolder(inflater, parent, R.layout.widgetlist_slideritem, connection, colorMapper),
         SeekBar.OnSeekBarChangeListener {
         private val seekBar: SeekBar = itemView.findViewById(R.id.seekbar)
         private var boundWidget: Widget? = null
 
         init {
             seekBar.setOnSeekBarChangeListener(this)
+            val date = SimpleDateFormat("MM-dd", Locale.US).format(Date())
+            if (date == "10-31") {
```

Comment:
```
val now = Date()
if (now.dayOfMonth == 31 && now.month == Month.OCTOBER) {
```

Reply:
These functions are deprecated, but I the recommended alternative.